docs/libFuzzer-integration.md

```diff
@@ -0,0 +1,179 @@
+# LibFuzzer Integration
+
+*** note
+**Note:** The libFuzzer integration is actively being developed and is
+experimental.
+***
+
+GraphicsFuzz can be used with [libFuzzer](http://llvm.org/docs/LibFuzzer.html)
+as a [custom mutator](https://cs.chromium.org/chromium/src/third_party/libFuzzer/src/FuzzerInterface.h)
+for fuzzing that is both coverage-guided and [structure-aware](https://github.com/google/fuzzer-test-suite/blob/master/tutorial/structure-aware-fuzzing.md).
+
+## Using the Integration
+
+1. Build GraphicsFuzz.
+
+2. Start the GraphicsFuzz CustomMutatorServer with the following command:
+
+```bash
+java -ea -cp graphicsfuzz/target/graphicsfuzz/jar/tool-1.0.jar com.graphicsfuzz.generator.tool.CustomMutatorServer
+Listening on port: 8666
+```
+
+3. Build a fuzz target (in a new shell, since the previous example won't
+   terminate). An example is provided below:
+
+```bash
+cat << EOF > fuzzer.cc
```

Comment:
Maybe this code should live in its own file and I should instruct people to download it?
What do you think?

Reply:
Done.